docs/logical_data_model.md

```diff
@@ -154,7 +154,7 @@ class ActivityReport {
   startDate : date
   activityRecipientType : string
   requester : string
-  * status : string
+  * overviewStatus : string
```

Comment:
This can be removed and computed in the client side if we choose to remove it.

Reply:
I agree. From looking at the issue, I don't think we'd need a field for this. I think the API could provide the necessary information as if it were part of the AR model itself.

Reply:
status will remain as is for legacy reasons, data can be transformed in a new ticket after multiple approvers is implemented